server/src/main/java/org/elasticsearch/cluster/metadata/IndexTemplateMetaData.java

```diff
@@ -32,28 +31,16 @@
 import org.elasticsearch.common.compress.CompressedXContent;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
-import org.elasticsearch.common.logging.DeprecationLogger;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.util.set.Sets;
-import org.elasticsearch.common.xcontent.ToXContent;
-import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.XContentFactory;
-import org.elasticsearch.common.xcontent.XContentHelper;
-import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.common.xcontent.*;
```

Comment:
`.*` imports aren't allowed in Elasticsearch code, which is why the test run failed - I suspect this is a change your IDE made automatically.

You can find guidance on how to configure your IDE to use explicit imports instead of wildcard imports [in our Contributing guide](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md) in the "Java Language Formatting Guidelines" section.